products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,380 @@
+which is known as a transfer fee. When a player moves from one team to another,
+their old contract is terminated and they negotiate a new one with the team
+they are moving to. In some cases, however, transfers can function similarly
+to player trades, as teams can offer another player on their team as part of the fee.
+
+As you may presume, there is a lot of money involved in the game of transfers. According to FIFA,
+in 2018, from January till September, there were 15,626 international transfers
+with fees totaling US$ 7.5 billion dollars.
+
+Football season is that part of the year during which football matches are held. A typical football
+season is generally from August/September to May, although in some countries, such as Northern Europe
+or East Asia, the season starts in the spring and finishes in autumn due to weather conditions encountered
+during the winter. 
+
+
+### Data Model
+
+In this article, we will present a graph model of football transfers 
+from season 1992/1993 to season 2019/2020 in following five leagues:
+* English Premier League
+* French Ligue 1
+* German Bundesliga
+* Italian Serie A
+* Spanish Primera Division
+
+The model consists of the following nodes:
+* `Team` - a football team with a property `name` (e.g. `"FC Barcelona"`).
+* `Player` - a professional football player, contains properties `name` (e.g. `"Luka Modric"`)
+and `position` (e.g. `"Central Midfield"`).
+* `League` - a football league where multiple teams play in, contains one property
+`name` (e.g. `"Premier League"`).
+* `Transfer` - represents football transfer that connects a `Player` 
+that is transferred from one `Team` to another `Team`
+within a `Season`. Transfer contains one optional property `fee` (e.g. `80.50`)
+that represents a transfer fee in millions of euros and one regular property `year` (e.g `1995`)
+that represents how old was a player when the transfer occurred.
+* `Season` - a football season with two properties `name` (e.g. `"2019/2020"`)
+and `year` (e.g. `2019`).
+
+Nodes are connected with the following edges:
+
+* `: TRANSFERRED_FROM` - connects team node `Team` to node `Transfer` representing a team
+where the player is being transferred from.
+* `: TRANSFERRED_TO` - connects node `Transfer` to team node `Team` where player is being transferred to.
+* `: TRANSFERRED_IN` - connects player node `Player` to node `Transfer` representing a player
+that was transferred in the connected transfer.
+* `: HAPPENED_IN` - connects node `Transfer` to the node `Season` in which transfer has happened.
+* `: PLAYS_IN` - connects node `Team` that plays in league node `League`.
+
+
+![](../data/football_transfers_metahraph.png)
+
+### Importing the Snapshot
+
+We have prepared a database snapshot for this example, so you can easily
+import it when starting Memgraph using the `--data-directory` option.
+
+```plaintext
+sudo -u memgraph \
+  /usr/lib/memgraph/memgraph --data-directory /usr/share/memgraph/examples/TEDTalk \
```

Comment:
```suggestion
  /usr/lib/memgraph/memgraph --data-directory /usr/share/memgraph/examples/FootballTransfers \
```

I would say we can now replace this to `FootballTransfers` because that's where we will save cypher commands.

Reply:
:+1: